### DISCUSSION TITLE
Orange icon in project [Apr 02]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1635] [Discord Threads]
> 🙋‍♂️ **liuq**  
## Hi everyone, I connected a local MySQL database, but displayed a orange status, it means loading metadata?  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/959500997925945384/unknown.png)

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### Probably project has failed, I see you 've given IP address as project name  
#  
> 💁 **o1lab**  
#### please change it to an alphanumeric character  
#  
> 🙋‍♂️ **liuq**  
#### thank you, I try it now  
#  
> 🙋‍♂️ **liuq**  
#### It has nothing to do with the name  
#  
> 💁 **o1lab**  
#### was it a graphql project ?   
####    
#### please see for logs in docker  
#  
> 💁 **o1lab**  
#### and raise an issue with details  
#  
> 🙋‍♂️ **liuq**  
#### I need setup a instance with same local db version?  
#  
> 🙋‍♂️ **liuq**  
#### mysql  
#  
> 🙋‍♂️ **liuq**  
#### Thank you, I saw docker logs `Access denied for user 'root'@'localhost' (using password: NO)`  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/959500999406530610)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
